src/hooks/useEditorView.ts

```diff
@@ -59,70 +36,86 @@ function withFlushedDispatch(
  * All state and props updates are executed in a layout effect.
  * To ensure that the EditorState and EditorView are never out of
  * sync, it's important that the EditorView produced by this hook
- * is only accessed through the `useEditorViewEvent` and
- * `useEditorViewLayoutEffect` hooks.
+ * is only accessed through the provided hooks.
  */
 export function useEditorView<T extends HTMLElement = HTMLElement>(
   mount: T | null,
-  props: EditorProps
-): EditorView | null {
-  const [view, setView] = useState<EditorView | null>(null);
-  const forceUpdate = useForceUpdate();
-
-  const editorProps = withFlushedDispatch(props, forceUpdate);
+  props: Props
+): EditorContextValue {
```

Comment:
The new return value here is a breaking change to the hook, so I've made this a minor version bump. I don't expect this will be troubling to anyone. Every use we have internally is through the ProseMirror component, and not through the hook directly.